src/components/recommendation.tsx

```diff
@@ -48,37 +50,104 @@ export function Recommendation({
+              {chat.helloMessage}
+            </p>
+          </div>
+          {chat.supervisors?.map((supervisor) => (
+            <Supervisor
+              key={supervisor.uuid}
+              supervisor={supervisor}
+              prompt={chat.prompt}
+              chatUuid={chat.uuid}
+            />
+          ))}
+        </>
+      )}
     </div>
   );
 }
+
+const loadingMessages = [
+  "Przeglądam bazę promotorów szybciej niż studenci dysk przed sesją...",
+  "Obliczam prawdopodobieństwo dostępności promotora z dokładnością do trzeciego miejsca po przecinku...",
+  "Szukam promotora tak intensywnie, jak student wolnego miejsca na parkingu uczelni...",
+  "Wykonuję całkowanie po zbiorze promotorów. Może zająć tyle co wykład z analizy...",
+  "Optymalizuję wybór promotora metodą najmniejszych studentów...",
+  "Przetwarzam dane szybciej niż student biegnący na ostatni pociąg...",
+  "Przeprowadzam transformatę Fouriera na liście promotorów. Może potrwać tyle co kompilacja Javy...",
+  "Pracuję ciężej niż student na dzień przed deadlinem...",
+  "Szukam idealnego promotora jak student działającego długopisu podczas kolokwium...",
+  "Wykonuję obliczenia kwantowe na procesorze studenckim zasilanym kawą...",
+  "Przeprowadzam symulację Monte Carlo twojej przyszłej współpracy z promotorem...",
+  "Obliczam gradient zainteresowań naukowych z krokiem uczenia się równym liczbie kawy w dziekanacie...",
+  "Generuję odpowiedź szybciej niż student wymówki o niedziałającym internecie...",
+  "Wykonuję zapytanie rekurencyjne do bazy wiedzy. Głębokość rekurencji: piwo^3...",
+  "Implementuję algorytm genetyczny krzyżujący najlepsze cechy promotorów...",
+  "Buduję sieć neuronową do klasyfikacji promotorów. Warstwa ukryta: prokrastynacja...",
+  "Przeprowadzam redukcję wymiarów macierzy promotorów metodą PCA (Promotor Component Analysis)...",
+  "Optymalizuję funkcję celu: minimalizacja liczby poprawek w pracy dyplomowej...",
+  "Wykonuję normalizację danych według skali studenckich ambicji...",
+  "Uruchamiam procedurę znajdowania promotora metodą prób i błędów (głównie błędów)...",
+  "Przetwarzam dane w czasie rzeczywistym (czyli wolniej niż internet w akademiku)...",
+  "Przetwarzam dane szybciej niż schodzi piwo na juwenaliach...",
+  "Pracuję nad tym ciężej niż nad usprawiedliwieniem nieobecności na laborce...",
+  "Kalkuluję prawdopodobieństwo znalezienia promotora, który nie sprawdza cytowań...",
+  "Ładowanie trwa dłużej niż trzeźwienie przed egzaminem...",
+  "Wyszukuję promotora, który nie będzie pytał o postępy co tydzień...",
+  "Optymalizuję czas odpowiedzi jak student optymalizuje liczbę nieobecności...",
+  "Analizuję dane tak dokładnie jak prowadzący sprawdza obecność na wykładzie...",
+  "System myśli tak trzeźwo jak student w piątkowy wieczór...",
+];
+function getRandomMessage() {
+  return loadingMessages[Math.floor(Math.random() * loadingMessages.length)];
+}
```

Comment:
yeah, they're cringy af (maybe because they're 🤖generated)
I've thought that it'd be a task for soft skill person to edit the texts or/and to create some texts that would be specific for PWr